client/components/tinymce/plugins/simple-payments/dialog/product-image-picker.jsx

```diff
@@ -0,0 +1,129 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+import AsyncLoad from 'components/async-load';
+import { getMediaItem } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import MediaActions from 'lib/media/actions';
+import MediaLibrarySelectedData from 'components/data/media-library-selected-data';
+import ProductImage from './product-image';
+import RemoveButton from 'components/remove-button';
+
+class ProductImagePicker extends Component {
+	static propTypes = {
+		featuredImage: PropTypes.object,
+		siteId: PropTypes.number,
+		translate: PropTypes.func,
+	};
+
+	state = {
+		isSelecting: false,
+	};
+
+	showMediaModal = () => {
+		const { siteId, featuredImage } = this.props;
+
+		if ( featuredImage ) {
+			MediaActions.setLibrarySelectedItems( siteId, [ featuredImage ] );
+		}
+
+		this.setState( { isSelecting: true } );
+	};
+
+	hideMediaModal = () => this.setState( { isSelecting: false } );
+
+	setImage = value => {
+		this.hideMediaModal();
+
+		if ( ! value ) {
+			return;
+		}
+
+		this.props.input.onChange( value.items[ 0 ].ID );
+	};
+
+	removeCurrentImage = event => {
+		event.stopPropagation();
+
+		this.props.input.onChange( false );
+	};
+
+	getImagePlaceholder() {
+		return (
+			<div className="dialog__product-image-placeholder">
+				<Gridicon icon="add-image" size={ 36 } />
+				{ this.props.translate( 'Add an Image' ) }
+			</div>
+		);
+	}
+
+	getCurrentImage() {
+		const { siteId } = this.props;
+
+		return (
+			<div className="dialog__product-image">
+				<ProductImage siteId={ siteId } imageId={ this.props.input.value } showEditIcon={ true } />
+				<RemoveButton onRemove={ this.removeCurrentImage } />
+			</div>
+		);
+	}
+
+	render() {
+		const { siteId, translate } = this.props;
+
+		if ( ! siteId ) {
+			return;
+		}
+
+		return (
+			<div>
```

Comment:
Let's give this a classname to match the file. Makes it easier to reverse search in code.

Reply:
Updated in 2143e538ba4897342daf11d7d0240793291ae50d